Beis.LearningPlatform.Web/Views/Shared/DiagnosticToolFormViews/_PartialRenderDiagnosticTool.cshtml

```diff
@@ -45,12 +45,21 @@
                                 </h2>
                                 <div class="govuk-error-summary__body">
                                     <ul class="govuk-list govuk-error-summary__list">
-                                        @foreach (var validationError in Model.validationErrors)
+                                        @if (((int)Model.FormType) > 2)
                                         {
-                                            <li class="@validationError.displayControlClass">
-                                                <a id="link-diagnostictoolformviews-renderdiagnostictool-validation-error-@(validationError.htmlId)" href="#@validationError.htmlId">@Html.Raw(validationError.errorMessage)</a>
+                                            <li>
+                                                <a href="#">Answer all of the questions below to continue</a>
```

Comment:
This a has no link?

Reply:
Design requires style to be same as a href style. This is used throughout the site for all validation messages for example. It could be changed so we have a new css style for this but that should then be applied throughout the site to be consistent. For long pages we use this as bookmark. But these pages are short.